### ISSUE TITLE
Bug with anki 2.1.35

### ISSUE THREAD

**Issue by aPaci95** [BlueGreenMagick/Edit-Field-During-Review-Cloze#38]
I get this everytime i edit something for the first time while i review, then after i dismiss the popup window it works fine for a while and it breaks again after some time (if for example i exit to the deck browser and i start reviewing again).

```
Debug info:
Anki 2.1.35 (84dcaa86) Python 3.8.0 Qt 5.14.2 PyQt 5.14.2
Platform: Windows 10
Flags: frz=True ao=True sv=2
Add-ons, last update check: 2020-11-04 18:52:04

Caught exception:
Traceback (most recent call last):
  File "aqt\webview.py", line 37, in cmd
  File "aqt\webview.py", line 123, in _onCmd
  File "aqt\webview.py", line 547, in _onBridgeCmd
  File "<C:\Users\pacia\Desktop\AnkiFolder\Anki21-35\decorator.pyc:decorator-gen-68>", line 2, in linkHandler_wrapper
  File "anki\hooks.py", line 638, in decorator_wrapper
  File "anki\hooks.py", line 635, in repl
  File "C:\Users\pacia\AppData\Roaming\Anki2\addons21\385888438\reviewer.py", line 193, in myLinkHandler
    saveThenRefreshFld(reviewer, note, fld, new_val)
  File "C:\Users\pacia\AppData\Roaming\Anki2\addons21\385888438\reviewer.py", line 153, in saveThenRefreshFld
    saveField(note, fld, new_val)
  File "C:\Users\pacia\AppData\Roaming\Anki2\addons21\385888438\reviewer.py", line 135, in saveField
    txt = Editor.mungeHTML(None, txt)
  File "aqt\editor.py", line 423, in mungeHTML
  File "aqt\gui_hooks.py", line 1606, in __call__
  File "aqt\editor.py", line 1225, in reverse_url_quoting
AttributeError: 'NoneType' object has no attribute 'mw'
```

**Comment by loveAnkisomuch** [comment]
me2

**Comment by BlueGreenMagick** [comment]
This has been fixed in the latest version with #37. Sorry for the long wait!
